h/models/es_sync_task.py

```diff
@@ -0,0 +1,19 @@
+from sqlalchemy import Column, Integer, text
+
+from h.db import Base
+from h.db.types import URLSafeUUID
+
+
+class EsSyncTask(Base):
+    """Task queue for the Postgres -> Elasticsearch synchronization task."""
+
+    __tablename__ = "es_sync_task"
+
+    #: Auto-incrementing unique ID for the row.
+    id = Column(Integer, primary_key=True)
+
+    #: The ID of the annotation to be synchronized to Elasticsearch.
+    annotation_id = Column(URLSafeUUID, nullable=False)
+
+    #: How many attempts that have been made to complete this task.
+    attempts = Column(Integer, server_default=text("0"))
```

Comment:
I think I'm gonna leave out priority for now. It isn't needed until we have messages being added to the queue for multiple different reasons, and when we do come to do that I'm not 100% sure that a simple priority int is going to be enough. If we just have integer priorities and the task consumes messages high-priority first then the lower-priority messages might never get consumed since the high-priority ones are coming in all the time. So we might want something like multiple periodic tasks, one for each priority. We might end up still wanting an integer priority column in the end but I think we should add it at the time as we'll know better then